test/functional/admin/statistics_announcement_publications_controller_test.rb

```diff
@@ -69,7 +75,7 @@ class Admin::StatisticsAnnouncementPublicationsControllerTest < ActionController
 
     assert_response :success
     has_search_results_table
-    assert_select "ul.govuk-error-summary__list a", text: "Publication type does not match: must be statistics"
+    assert_select "ul.govuk-error-summary__list a", text: "Publication type does not match announcement type: must be 'Official Statistics'"
```

Comment:
To single quote or not to singe quote. I think Rosa once showed me some messaging with quotes and now it's how I do things. But they looked horrific on the inset text so no quotes there.

Reply:
🤷 LGTM